frontend/src/types.ts

```diff
@@ -418,7 +426,7 @@ export interface TeamType extends TeamBasicType {
         | { recordHeaders?: boolean; recordBody?: boolean }
         | undefined
         | null
-    session_replay_config: { record_canvas?: boolean } | undefined | null
+    session_replay_config: { record_canvas?: boolean; ai_config?: SessionRecordingAIConfig } | undefined | null
```

Comment:
Not for this PR but we should be conscious that `session_replay_config` could end up with a lot of keys - even ones we end up not needing going forward. Might not be an issue but could get annoying / confusing over time to have data we no longer rely on